packages/ui/src/council/queries/council.graphql

```diff
@@ -379,3 +379,16 @@ query GetPastCouncilStats($startBlock: Int!, $endBlock: Int!) {
     ...CouncilSpendingEventFields
   }
 }
+
+query GetCouncilorElectionEvent($lockAccount: String) {
+  memberships(where: { controllerAccount_eq: $lockAccount }, orderBy: [createdAt_DESC], limit: 1) {
+    councilMembers {
```

Comment:
Unlike with `memberInvitedEvents` the `controllerAccount` might not be the staking account here:
```suggestion
  councilMembers (where: { stakingAccountId_eq: $lockAccount }, orderBy: [createdAt_DESC], limit: 1) {
```